Native Linux builds now fail with this PR:
```
-- Found FriBidi: /usr/lib/x86_64-linux-gnu/libfribidi.so (found version "0.19.7")
-- Found PCRE: /usr/lib/x86_64-linux-gnu/libpcrecpp.so (found version "8.38")
CMake Error at /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:148 (message):
  Could NOT find RapidJSON (missing: RapidJSON_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:388 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/FindRapidJSON.cmake:22 (find_package_handle_standard_args)
  cmake/scripts/common/Macros.cmake:315 (find_package)
  CMakeLists.txt:169 (core_require_dep)


-- Configuring incomplete, errors occurred!
See also "/home/neil/projects/kodi.git/CMakeFiles/CMakeOutput.log".
See also "/home/neil/projects/kodi.git/CMakeFiles/CMakeError.log".
```
I do not have RapidJSON installed on the host.

Without this PR:
```
-- Could NOT find RAPIDJSON (missing:  RapidJSON_INCLUDE_DIR)
```
So for some reason it's not downloading from the mirror...

--- a/cmake/modules/FindRapidJSON.cmake
+++ b/cmake/modules/FindRapidJSON.cmake
@@ -10,17 +10,22 @@
 #
 
 if(PKG_CONFIG_FOUND)
-  pkg_check_modules(PC_RapidJSON RapidJSON QUIET)
+  pkg_check_modules(PC_RapidJSON RapidJSON>=1.0.2 QUIET)
+endif()
+
+if(CORE_SYSTEM_NAME STREQUAL windows OR CORE_SYSTEM_NAME STREQUAL windowsstore)
+  set(RapidJSON_VERSION 1.0.2)
+else()
+  set(RapidJSON_VERSION ${PC_RapidJSON_VERSION})
 endif()
 
 find_path(RapidJSON_INCLUDE_DIR NAMES rapidjson/rapidjson.h
                                 PATHS ${PC_RapidJSON_INCLUDEDIR})
 
-set(RapidJSON_VERSION ${PC_RapidJSON_VERSION})
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(RAPIDJSON
-                                  REQUIRED_VARS RapidJSON_INCLUDE_DIR
+find_package_handle_standard_args(RapidJSON
+                                  REQUIRED_VARS RapidJSON_INCLUDE_DIR RapidJSON_VERSION
                                   VERSION_VAR RapidJSON_VERSION)
 
 if(RAPIDJSON_FOUND)

